rakelib/test.rake

```diff
@@ -87,7 +87,7 @@ namespace "test" do
     require "simplecov"
     SimpleCov.start do
       # Skip non core related directories and files.
-      ["vendor/", "spec/", "bootstrap/rspec", "Gemfile", "gemspec"].each do |pattern|
+      ["vendor/", "test/", "bootstrap/rspec", "Gemfile", "gemspec"].each do |pattern|
```

Comment:
Why are we now calling our rspec specs 'test'. Isn't that lesson common for rspec?